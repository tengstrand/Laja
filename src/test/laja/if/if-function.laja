
#function thisIsTrue()
  #return (true)
#end

#function booleanToInt()
  #if (true)
    #return (1)
  #else
    #return (2)
  #end
#end

#write "target/test-output/if-function.out"
  #if (thisIsTrue())
This is true!
  #end
{booleanToInt()}
#end
