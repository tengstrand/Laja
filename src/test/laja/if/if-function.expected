This is true!
1
